website/public/admin.py

```diff
@@ -1,3 +1,15 @@
-# from django.contrib import admin
+from django.contrib import admin
```

Comment:
Is this a good place for this or should I have used another `admin.py` file? 🤔

Reply:
Good question, we're using the `public` app as a source of things that should be accessible to the public without logging into the website so FlatPages-related stuff seems like it should fit well here (the "admin" stuff is never accessible to the public anyways)